### DISCUSSION TITLE
Taint analysis configuration and results

### DISCUSSION THREAD

**Discussion by mmvpm** [UnitTestBot/UTBotJava#1312] [Ideas]
1. We need to develop a way to configure utbot taint analysis:
    - Sources
    - Passthroughs
    - Cleaners
    - Sinks
2. Also we want to display the results to the user on the Problems tab. So we need to decide what the information will be shown:
    - a short message describing the detected problem
    - corresponding execution trace

**Comment by Difocd** [comment]
Are there any taint analysis implementations or plans in UTBotJava now? I can not find any.

**Reply by korifey** [reply]
There are certainly plans for this. Currently there is branch “taint-analysis” with prototype that works pretty good.

**Reply by Difocd** [reply]
Thank you for your quick reply. I'm going to check the code.
